dftimewolf/lib/logging_utils.py

```diff
@@ -102,4 +106,11 @@ def format(self, record: LogRecord) -> str:
       if loglevel_color:
         message = loglevel_color + message + RESET_SEQ
       record.msg = message
+
+    if self.threaded:
+      stack = [i[3] for i in inspect.stack()]
```

Comment:
What's `i[3]` here? Is there another attribute we can access that would be more clear to use in the code?